study.md

```diff
@@ -161,15 +164,18 @@ What is a response? what does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+A response is something sent to a server and contains rescource representatiion
+that has information that is dependent upon the application, it can also be
+represented in multiple different ways by the server. They are documents that
+are returned as part of the response body.
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+curl GET Reddit.com 
 ```
```

Comment:
`curl https://www.reddit.com`

among other ways you need the full url so it has the http, the full path and while the get is accurate, just a curl with a valid url will return the html of that url.